src/main/resources/application.properties

```diff
@@ -184,7 +192,7 @@ ccd.conditional-apis.case-assigned-users-and-roles.enabled=${CCD_CONDITIONAL_API
 security.logging.filter.path.regex=${DATA_STORE_SECURITY_LOGGING_FILTER_PATH_REGEX:.*}
 
 # CCD access control
-enable-attribute-based-access-control=${ENABLE_ATTRIBUTE_BASED_ACCESS_CONTROL:false}
+enable-attribute-based-access-control=${ENABLE_ATTRIBUTE_BASED_ACCESS_CONTROL:true}
```

Comment:
Is changing this default the reason for so many pipeline IT failures? Was this intentional as part of this PR?